### PR TITLE
Add Acknowledgments

### PR DESCRIPTION
**Pull request by awoods**
Resolves: https://github.com/fcrepo/fcrepo-specification/issues/268

### CODE REVIEW

**Review by escowles — DISMISSED** [review]

File: index.html [hunk 1]
```diff
@@ -1277,9 +1277,14 @@ <h2>Security Considerations</h2>
...
 
-    <section class='appendix'>
+    <section id="acknowledgments" class="informative appendix">
       <h2>Acknowledgments</h2>
       <p>
+        The following people have been instrumental in providing thoughts, feedback, reviews, content, criticism and
+        input in the creation of this specification:
+      </p>
+      <p style="margin-left: 3em;">
+        Sarven Capadisli, Herbert Van de Sompel, the Fedora Repository, Islandora and Samvera communities.
```
Comment (zimeon) [inline index.html]: The communities aren't people, and there is an ambiguity in the list as to which elements "communities" applies to. Perhaps:

> Sarven Capadisli, Herbert Van de Sompel, **and members of** the Fedora Repository, Islandora and Samvera communities.

**Review by zimeon — CHANGES_REQUESTED** [review]

**Review by dannylamb — DISMISSED** [review]
Conditionally approving once @zimeon's feedback is addressed.

**Review by escowles — APPROVED** [review]

**Comment by awoods** [comment]
@zimeon : changes made:
https://github.com/fcrepo/fcrepo-specification/pull/272/commits/ce382bdc32f9566cc8813812ed73c87b74dcf48e
..based on comment: 
https://github.com/fcrepo/fcrepo-specification/pull/272#pullrequestreview-78576610

**Review by zimeon — APPROVED** [review]

**Review by dannylamb — APPROVED** [review]
:shipit:

**Pull request merged**